spec/common/participants.html

```diff
@@ -1 +1 @@
-Members of the RDF Dataset Canonicalization and Hash Working Group Group included Ahamed Azeem, Ahmad Alobaid, Andy Seaborne, Benjamin Goering, Brent Zundel, Dan Brickley, Dan Yamamoto, Dave Longley, David Lehn, Gregg Kellogg, Ivan Herman, Jesse Wright, Kazue Sako, Leonard Rosenthol, Mahmoud Alkhraishi, Manu Sporny, Markus Sabadello, Michael Prorock, Phil Archer, Pierre-Antoine Champin, Sebastian Crane, Ted Thibodeau, Timothée HAUDEBOURG, and Tobias Kuhn.
+Members of the RDF Dataset Canonicalization and Hash Working Group Group included Ahamed Azeem, Ahmad Alobaid, Benjamin Goering, Brent Zundel, Dan Brickley, Dan Yamamoto, Dave Longley, David Lehn, Gregg Kellogg, Ivan Herman, Jesse Wright, Kazue Sako, Leonard Rosenthol, Mahmoud Alkhraishi, Manu Sporny, Markus Sabadello, Michael Prorock, Phil Archer, Pierre-Antoine Champin, Sebastian Crane, Ted Thibodeau, Timothée Haudebourg, and Tobias Kuhn.
```

Comment:
We could update the script to generate "At the time of publication, members of the ...". Or, if there is an API endpoint that includes both current and former members, we could use that.

Reply:
It seems that we can use `former` flag to include both current and former members:
https://api.w3.org/groups/wg/rch/users?former=true

Reply:
@yamdan thanks for the pointer; I added it to the script and regenerated the participants.